scripts/broken-links-slack-payload.mjs

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-console */
+import fs from 'fs'
+import core from '@actions/core'
+import { program } from 'commander'
+
+const DEFAULT_INPUT_FILE = 'broken-links-report.json'
+const DEFAULT_OUTPUT_FILE = 'broken-links-slack-payload.json'
+const GROUP_TO_NOTIFY = 'subteam^S010U41C30V|cms-helpdesk' // '<!subteam^S010U41C30V|cms-helpdesk>'
+const RUN_URL = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}`
+
+// Gather any command line options or provide defaults
+program
```

Comment:
I used commander to process command-line flags. I realize this isn't consistent with other scripts. Can convert this to use env vars if we feel strongly.

Reply:
the env-loader package also uses commander to process cli flags into env vars, i'm fine with doing this separately here because it's a separate thing